tests/test_wave_05.py

```diff
@@ -80,34 +78,39 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title"
+    })
 
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    assert response.status_code == 204
+    
+    query = db.select(Goal).where(Goal.id == 1)
+    goal = db.session.scalar(query)
+    
+    assert goal.title == "Updated Goal Title"
 
+    # DO I NEED TO ADD ANOTHER ASSERTION -- it said 3
```

Comment:
You could also check to ensure that nothing changed about the `.tasks` attribute.

Reply:
But I think the two you have there are fine. @ellenjin